$OpenBSD: patch-sys-uuid_cc,v 1.3 2019/08/08 21:21:17 juanfra Exp $

g++ 8.3 on sparc64:

"sys-uuid.cc:96:12: note: 'strlen' is defined in header '<cstring>'; did you forget to '#include <cstring>'?"

"sys-uuid.cc:97:5: note: 'strcpy' is defined in header '<cstring>'; did you forget to '#include <cstring>'?"

Index: sys-uuid.cc
--- sys-uuid.cc.orig
+++ sys-uuid.cc
@@ -68,6 +68,7 @@ void uuid_unparse_lower(uuid_t &uu, char *out)
 
 #include <cassert>
 #include <cerrno>
+#include <cstring>
 #include <cstdint>
 
 #include <uuid.h>
